site/docs/encoding/legend.md

```diff
@@ -61,7 +61,7 @@ To customize legends, a `legend` object in [an encoding channel's definition](en
 
 {:#properties}
 
-{% include table.html props="type,entryPadding,format,offset,orient,padding,tickCount,title,values,zindex" source="Legend" %}
```

Comment:
So entryPadding is removed?  (Sounds like a breaking change.)

Reply:
Yes